### DISCUSSION TITLE
[Feature] Add Matrix

### DISCUSSION THREAD

**Discussion by Ahwxorg** [immich-app/immich#6492] [Feature Request]
### The feature

**What:**
It might be nice to add a Matrix group chat for support, could maybe be bridged with Discord.

**Reason**: I do not use Discord, and wanted to ask a support question, and I don't want to sacrifice my privacy to Discord just to ask a question about a open source project. Thought I'd ask, since you have `Privacy should not be a luxury` on your website.

**Comment by alextran1502** [comment]
You don't have to use Discord to get support. You can have questions directly on GitHub as well. Discord is just an easy access for most people that wants to ask questions

**Reply by Ahwxorg** [reply]
Alright, thanks for answering :)

**Reply by Ahwxorg** [reply]
I don't know why I didn't pick up on this earlier, but GitHub isn't really perfect either, Matrix with a Discord bridge would be more ideal (but then again Github is already a lot better than Discord, soooo...), or just plain old email.

**Reply by CEbbinghaus** [reply]
would like to reinforce that is is just as relevant today, Discord is not discoverable and while by default matrix isn't either [there are projects aiming to change that](https://github.com/matrix-org/matrix-viewer). 

It also more closely aligns with the want to self host and be independent of big companies owning our data, so why not let people phone in directly from their homeserver via the matrix protocol.

**Comment by soaska** [comment]
Yep, it would be nice to make a matrix room with discord bridge

**Comment by schildbach** [comment]
I'd appreciate a Matrix room/space as well. Many technical projects migrated from proprietary, centralized Discord to the decentralized and free-software Matrix.

**Comment by TuxEnthusiast** [comment]
+1
